core/src/script/CGXP/widgets/tree/LayerTree.js

```diff
@@ -404,6 +411,17 @@ cgxp.tree.LayerTree = Ext.extend(Ext.tree.TreePanel, {
 
                 var timeWidget;
                 if (item.time) {
+                    if (this.initialState && this.initialState['time_' + item.name]) {
+                      var times = this.initialState['time_' + item.name].split('/');
+                      if (times.length === 2) {
```

Comment:
Yes, probably, but there's not example like this in the demo
I will check this, thanks.